CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 1.1.1
+
+* Add swift4 compatibility.  
```

Comment:
You should mention something like: "Now the `swift3` template is compatible `swift4` too, so you can use `-t swift3` even for Swift 4"

Reply:
Ok I will this